src/main/java/teammates/ui/controller/InstructorFeedbacksPageData.java

```diff
@@ -160,9 +164,13 @@ private FeedbackSessionsForm buildBasicForm(List<CourseAttributes> courses, Stri
         List<ElementTag> courseIdOptions = getCourseIdOptions(courses, courseIdForNewSession, instructors, newFeedbackSession);
         boolean isSubmitButtonDisabled = courseIdOptions.isEmpty();
         
-        // adds the placeholder option to courseIdOptions if it is empty
-        if (isSubmitButtonDisabled) {
-            addPlaceholder(courseIdOptions);
+        // Adds a placeholder option to courseIdOptions if it is empty
+        if (courseIdOptions.isEmpty()) {
+            if (courses.isEmpty())
+                addPlaceholder(courseIdOptions, Const.StatusMessages.INSTRUCTOR_NO_ACTIVE_COURSES);
+            else {
+                addPlaceholder(courseIdOptions, Const.StatusMessages.INSTRUCTOR_NO_MODIFY_COURSES_SESSION_PERMISSION);
+            }
```

Comment:
I don't really like that code you have there too. It's not obvious why <code>addPlaceHolder</code> would need that much information.